### DISCUSSION TITLE
Windows download link is broken

### DISCUSSION THREAD

**Discussion by ghost** [git-lfs/git-lfs#4956] [General]
Link on Git-LFS download page links to 3.1.3, that link gives a 404. After some investigating, it looks like current release is at 3.1.4, though I'm not sure why the 3.1.3 release is gone. Just wanted to let someone know the page is OOD.

![image](https://user-images.githubusercontent.com/340268/164258938-f9a8cbbe-850e-45f2-9a32-0321243f97e7.png)

**Comment by bk2204** [comment]
Hey,

Thanks for letting us know.  3.1.3 never finished the release process because of a couple of problems where I didn't cherry-pick enough patches into the release, so we've done a 3.1.4, which includes all of the changes.  I've just pushed an updated commit, and the page should now be up to date.
